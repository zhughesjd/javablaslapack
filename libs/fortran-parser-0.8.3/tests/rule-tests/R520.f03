! Testing allocatable-stmt, R520
10  allocatable a
allocatable :: B( :,  :), C(:), scalar
end
