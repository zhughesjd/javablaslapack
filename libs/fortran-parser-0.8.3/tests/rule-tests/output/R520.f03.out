R1101:main-program__begin:
R527-F2008:allocatable-decl: object_name=[@7,49:49='a',<206>,2:16] hasArraySpec=false hasCoArraySpec=false
R 520:allocatable-stmt: label=[@2,33:34='10',<10>,2:0] keyword=[@5,37:47='allocatable',<67>,2:4] eos=[@8,50:50='\n',<4>,2:17] count=1
R 510:array-spec-element: type=705
R 510:array-spec-element: type=705
R 510:array-spec: count=2
R527-F2008:allocatable-decl: object_name=[@13,66:66='B',<206>,3:15] hasArraySpec=true hasCoArraySpec=false
R 510:array-spec-element: type=705
R 510:array-spec: count=1
R527-F2008:allocatable-decl: object_name=[@24,77:77='C',<206>,3:26] hasArraySpec=true hasCoArraySpec=false
R527-F2008:allocatable-decl: object_name=[@30,83:88='scalar',<206>,3:32] hasArraySpec=false hasCoArraySpec=false
R 520:allocatable-stmt: keyword=[@9,51:61='allocatable',<67>,3:0] eos=[@31,89:89='\n',<4>,3:38] count=3
R 204:specification-part: numUseStmts=0 numImportStmts=0 numDeclConstructs=2
R1103:end-program-stmt: id=null
R1101:main-program: hasProgramStmt=false hasExecutionPart=false hasInternalSubprogramPart=false
